docs/core-manage-asdf-vm.md

```diff
@@ -277,29 +315,226 @@ asdf update --head
 brew upgrade asdf
 ```
 
+### -- Pacman --
+
+Refer to the [Arch Pacman upgrade documentation](https://wiki.archlinux.org/index.php/pacman#Upgrading_packages).
+
 <!-- select:end -->
 
 ## Remove
 
-Uninstalling `asdf` is as simple as:
+To unistall `asdf` follow these steps:
 
-1.  In your `.bashrc`/`.bash_profile`/`.zshrc`/`config.fish` find the lines that source `asdf.sh` and the completions (this may be a ZSH Framework plugin). In Bash, the lines look something like this:
+<!-- select:start -->
+<!-- select-menu-labels: Operating System,Shell,Installation Method -->
 
-    ```shell
-    . $HOME/.asdf/asdf.sh
-    . $HOME/.asdf/completions/asdf.bash
-    ```
+### --Linux,Bash,Git--
 
-    Remove these lines and save the file.
+1. In your `~/.bashrc` remove the lines that source `asdf.sh` and the completions:
 
-2.  Run
+```shell
+. $HOME/.asdf/asdf.sh
+. $HOME/.asdf/completions/asdf.bash
+```
+
+2. Remove the `$HOME/.asdf` dir:
+
+```shell
+rm -rf ${ASDF_DATA_DIR:-$HOME/.asdf}
+```
+
+### --Linux,Fish,Git--
+
+1. In your `~/.config/fish/config.fish` remove the lines that source `asdf.sh`:
+
+```shell
+source ~/.asdf/asdf.fish
+```
+
+and remove completions with this command:
+
+```shell
+rm -rf ~/.config/fish/completions/asdf.fish
+```
+
+2. Remove the `$HOME/.asdf` dir:
+
+```shell
+rm -rf ${ASDF_DATA_DIR:-$HOME/.asdf}
+```
+
+### --Linux,ZSH,Git--
+
+1. In your `~/.zshrc` remove the lines that source `asdf.sh` and completions:
+
+```shell
+. $HOME/.asdf/asdf.sh
+
+# ...
+fpath=(${ASDF_DIR}/completions $fpath)
+autoload -Uz compinit
+compinit
+```
+
+**OR** the ZSH Framework plugin if used.
+
+2. Remove the `$HOME/.asdf` dir:
+
+```shell
+rm -rf ${ASDF_DATA_DIR:-$HOME/.asdf}
+```
+
+### --Linux,Bash,Pacman--
+
+1. In your `~/.bashrc` remove the lines that source `asdf.sh` and the completions:
+
+```shell
+. /opt/asdf-vm/asdf.sh
+```
+
+2. Uninstall with your package manager:
+
+```shell
+pacman -Rs asdf-vm
+```
+
+### --Linux,Fish,Pacman--
+
+1. In your `~/.config/fish/config.fish` remove the lines that source `asdf.fish`:
+
+```shell
+source /opt/asdf-vm/asdf.fish
+```
+
+2. Uninstall with your package manager:
+
+```shell
+pacman -Rs asdf-vm
+```
 
-    ```shell
-    rm -rf ${ASDF_DATA_DIR:-$HOME/.asdf} ~/.tool-versions
-    ```
```

Comment:
Still need this even on Arch, since all plugins and stuff are stored in `~/.asdf`.